path-manager/riff-raff.yaml

```diff
@@ -6,7 +6,7 @@ deployments:
     app: path-manager
     parameters:
       amiTags:
-        Recipe: editorial-tools-bionic-java8
+        Recipe: editorial-tools-bionic-java8-WITH-cdk-base
```

Comment:
is there any reason not to add cdk-base to the main recipe?

Reply:
Until all project use devx-logs @akash1810 explained that it would start fluentbit twice, so this is a temporary measure.

Reply:
I took a look at how things are configured in amigo, now I'm not sure I understand why - the cdk-base role depends on the fluentbit role, so if this is the case wouldn't this new recipe also have the same problem?

Reply:
@nicl confirmed its safe to use `cdk-base` on the main recipe (bake is running now), so I've put the `riff-raff.yaml` back to how it was and updated the PR description. once baked, I will check in CODE, before merging.